evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -0,0 +1,62 @@
+from jina.executors.evaluators.text import BaseTextEvaluator
+
+class BleuEvaluator(BaseTextEvaluator):
+    """
+    :class:`BleuEvaluator`Bilingual Evaluation Understudy Score. 
+    Evaluates the generated sentence (actual) against a desired sentence. 
+    It will use the Bleu on NLTK package.
+    A perfect match will score 1.0 and a complete mismatch will score 0.0
+
+    The NLTK library can score n-gram individually or cummulative.
+    Here we use the cumulative as it is more precise.
+    https://machinelearningmastery.com/calculate-bleu-score-for-text-python/
+
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+    @staticmethod
+    def get_score(actual_list, desired_list, n_gram):
```

Comment:
add type hinting, so that one can see what types are entering, they seem to be of the same type, but for the calling point of view seems that one is a list and the other a List of Lists